stdlib/2/typing.pyi

```diff
@@ -457,7 +457,8 @@ def cast(tp: str, obj: Any) -> Any: ...
 class NamedTuple(tuple):
     _fields = ...  # type: Tuple[str, ...]
 
-    def __init__(self, typename: str, fields: Iterable[Tuple[str, Any]] = ..., *,
+    def __init__(self, typename: AnyStr,
```

Comment:
I think you want `Union[str, unicode]`. AnyStr is a typevar, which does something different.

Reply:
Thanks, fixed.

I couldn't decide whether to use `str` or `bytes` and whether to use `unicode` or `Text` (the file is a bit inconsistent). Since this change affects only Python 2, I went with `str` and `unicode` route as it feels most "py2thonic".